src/check_utils.c

```diff
@@ -0,0 +1,36 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   check_utils.c                                      :+:      :+:    :+:   */
```

Comment:
Naming: all utils files are names utils_something. 

In this case should be utils_check.c